rubble/src/lib.rs

```diff
@@ -24,7 +24,7 @@
 #[macro_use]
 mod log;
 #[macro_use]
-mod utils;
+pub mod utils;
```

Comment:
Why was this required?

Reply:
I think I enabled it because I wanted to use hex slice in my application, I shouldn't have added that change. It does raise the question as to hexslice being split out as its own crate? It's tiny but pretty useful 🤷‍♀

Reply:
I think `HexSlice` and `Hex` are the 2 types that I reimplement most often, especially in embedded projects, so yeah, maybe it would make sense to put them in their own crate. But just having these 2 in there is barely more than 20 lines of code, so that crate should probably do a bit more to be useful.

Reply:
That's what I was thinking, we don't want to end up with a million NPM-esque micro packages.

Reply:
Would it hurt to make the utils module public?

Reply:
Maybe they can be added to the `hex` crate: https://docs.rs/hex

Reply:
> Would it hurt to make the utils module public?

I don't really want to provide API stability for that module. It's meant to be an implementation detail of Rubble. If you need `HexSlice`, feel free to copy it out though :)

Reply:
Ah good idea

Reply:
They're already discussing something like this in https://github.com/KokaKiwi/rust-hex/issues/8